docs/cow-protocol/tutorials/cow-explorer/order.md

```diff
@@ -40,37 +41,30 @@ Below is a simple example of Order ID [`56ac15a9`](https://explorer.cow.fi/order
 
 </details>
 
-A more complex example may be a user who has placed a _partially fillable_ order. This means that the user has expressed a swap intent for a certain amount of tokens, but the order may be filled in multiple batches. This can only be used with `Limit` orders.
+## Partially fillable
+
+A more complex example may be a user who has placed a _partially fillable_ order. This means that the user has expressed a swap intent for a certain amount of tokens, but the order may be filled in multiple batches. This can only be used with [Limit orders](../cow-swap/limit).
 
 ![Partially fillable order overview](/img/explorer/order_partial_overview.png)
 
+By clicking the 'View fills' button, the user can see the fills that have been executed for this order. Fills for the above example are show below:
+
 ![Partially fillable order fills](/img/explorer/order_partial_fills.png)
 
+Here we can see that the order has been 100% filled, but in dozens of batches. For ease of analysis, an overview is shown providing:
 
-#### States
+- The total amount filled (in this case 100%)
+- The average execution price (in this case 0.22134041 ANT for DAI)
+- Total surplus (in this case 0.39% / 860.894 DAI)
+- The limit price applied across all fills (in this case 0.22222222 ANT for DAI)
 
-As CPE is meant to help users visualize meta transactions (signed orders), a critical parameter to show is the State of the signed order. This parameter has the following status:
+## States
 
-- **Open** (pending)**:** State for a standing order. Orders in this state are being considered by the solvers. It is the entry state for all orders, the default from the moment the order is placed. From here the order can transition to all other states
+As CoW Explorer is meant to help users visualize their signed orders, a critical parameter to show is the _state_ of the signed order. This parameter has the following status:
+
+- **Open** (pending)**:** State for a standing order. Orders in this state are being considered by the solvers. It is the entry state for all orders, the default from the moment the order is placed. From here the order can transition to all other states.
```

Comment:
Would be worth to add a note about pre-signing orders?
For those types of orders, `Open` is not the entry state.

(Note, I don't know how to create foot notes, this is just a suggestion on the content)

```suggestion
- **Open** (pending)**:** State for a standing order. Orders in this state are being considered by the solvers. It is the entry state for all orders*, the default from the moment the order is placed. From here the order can transition to all other states.

^* Except for pre-sign orders, where the first state is [Pre-signing](link to pre-sign entry)
```

Reply:
Footnote added. Unfortunately can't link easily back up, but given that it's so close, should be OK.